src/main/java/ValidationMessages.properties

```diff
@@ -47,3 +48,5 @@ password.current=Please enter your current password.
 password.validate=Password reset page default email message.
 
 guestbook.name=Enter a name for the guestbook
+
```

Comment:
@sekmiller These extra returns at the bottom are not needed either.

Reply:
Ok. will do.